scripts/sync_repo_files.sh

```diff
@@ -19,11 +19,13 @@ if [ -z "${GITHUB_TOKEN}" ]; then
   exit 1
 fi
 
+# List of files that should be synced.
+SYNC_FILES="CODE_OF_CONDUCT.md LICENSE Makefile.common"
```

Comment:
LICENSE usually have a different year in it and cannot be simply copied.

Reply:
Oh, prometheus don't fill [those fields](https://github.com/prometheus/prometheus/blob/master/LICENSE#L189) so disregard my previous comment.